src/lib/ScriptUtils.js

```
@@ -0,0 +1,74 @@
+      if (scriptClass?.name !== className) {
+        continue;
+      }
+      const fields = scriptClass?.fields || [];
+      if (scriptClass?.fields.includes(field) || scriptClass?.fields.includes('*')) {
+        return true;
+      }
+      for (const currentField of fields) {
+        if (Object.prototype.toString.call(currentField) === '[object Object]') {
+          if (currentField.name === field) {
+            if (typeof currentField.validator === 'string') {
+              validator = eval(currentField.validator);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)